docs/reference/release-notes/highlights.asciidoc

```diff
@@ -30,6 +30,36 @@ Other versions:
 
 // tag::notable-highlights[]
 [discrete]
+
+=== SQL: Support for cross-cluster search
+
+Cross-cluster search (CCS) is now supported in SQL using the CCS-specific
+`cluster:index` notation where `cluster` maps to a SQL catalog and
+`index` to a table.
+
+=== Search: Node-level can-match action
+
+
```

Comment:
@ywelsch Do you mind helping provide a draft for this section?

I took a look at https://github.com/elastic/elasticsearch/pull/78765, but I wasn't familiar with the `can-match` action. If we include some more introductory information about that action and when ES may use it, I think we can use the rest of the description in the PR.